arbitrum-docs/for-devs/gentle-introduction-dapps.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "A gentle introduction: Building Arbitrum dApps"
+description: "Arbitrum is a suite of Ethereum scaling solutions that includes Arbitrum One, Arbitrum Nova, Stylus, and the Arbitrum Bridge. This guide introduces you to decentralized apps on Arbitrum: what they are, how they work, and the value that they provide."
+---
+
+import PublicPreviewBannerPartial from '../partials/_public-preview-banner-partial.md'; 
+
+<PublicPreviewBannerPartial />
+
+import GentleIntroPartial from '../partials/_gentle-intro-partial.md';
+
+<GentleIntroPartial />
```

Comment:
The `GentleIntroPartial`, as it stands, isn't _specifically about_ building dapps, so its content feels a little out of place here, but this is temporary. We can hide it with an "under construction" banner like the other gentle intro stubs, if we prefer to wait until the dev-specific gentle intro is ready.